tensorflow/training/docker/2.2.1/py3/cu101/Dockerfile.gpu

```diff
@@ -0,0 +1,228 @@
+FROM nvidia/cuda:10.1-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
```

Comment:
We will need different version streams for cu101 and cu102

Reply:
I'll remove this file to avoid confusion. we have different branch for cu101 PR